modules/ROOT/pages/migration-from-7x.adoc

```diff
@@ -3,9 +3,628 @@
+* [ ] Review and update third-party plugins
+* [ ] Test all custom event handling
+
+
+=== Plugin Updates
+
+==== Language Pack Filename Changes
+// #TINY-12090
+
+Language pack filenames have been standardized to follow the RFC5646 format. This update ensures consistent language handling across platforms and improves internationalization support. While both the legacy underscore format (e.g., `en_GB.js`) and the new hyphenated format (e.g., `en-GB.js`) are supported in {productname} {release-version}, the underscore format is deprecated and will be removed in {productname} 9. Migrating to the RFC5646 format now will ensure future compatibility and reduce maintenance overhead during upcoming upgrades.
+
+**Impact**: Custom language packs and configurations using the underscore-based format should update before upgrading to {productname} 9 to avoid loading failures.
+
+**Migration checklist:**
+
+* [ ] Identify all language pack files in your deployment
+* [ ] Rename files to RFC5646 format (e.g., `en_GB.js` → `en-GB.js`)
+* [ ] Update configuration references to language files
+* [ ] Update build scripts that handle language files
+* [ ] Test language switching in your application
+* [ ] Update custom translation files to use the new format
+
+[source, javascript]
+----
+// Deprecated format (supported only in versions 8.x)
+language_url: '/langs/en_GB.js'
+
+// Recommended format
+language_url: '/langs/en-GB.js'
+----
+
+[IMPORTANT]
+Support for the underscore format will be removed in {productname} 9. Early migration is recommended.
+
+==== Update to Image and Accessibility Checker Plugins
+// #TINY-12226
+
+The Image and Accessibility Checker plugins now follow the latest W3C standards for decorative images, requiring an empty alt attribute rather than a `+role="presentation"+` attribute. This change helps improve accessibility support.
+
+**Impact**: Customers using these plugins will need to update their configurations to ensure continued compliance with accessibility standards.
+
+**Migration checklist:**
+
+* [ ] Identify images previously using `role="presentation"` for decorative purposes in your content
+* [ ] Replace those with empty alt attributes `alt=""`
+* [ ] Update image plugin configuration if customized
+* [ ] Test accessibility checker with updated content
+
+For more information on the changes, see: xref:a11ychecker.adoc##image-rules[Accessibility Checker: Image rules].
+
+[[pagebreak-split-block-default]]
+==== Page Break Plugin: Export PDF/Word Compatibility and Option Updates
+// #TINY-12013, #TINY-12462
+
+The Page Break plugin has been updated to work out-of-the-box with the xref:exportpdf.adoc[Export to PDF] and xref:exportword.adoc[Export to Word] plugins, addressing a common pain point for developers. As part of this update:
+
+* The default value of the xref:pagebreak.adoc#pagebreak_separator[`pagebreak_separator`] option now uses a block tag (`<div style="break-after: page
```

Comment:
The separator change should be the focus of the breaking change, not split block. Add a new `pagebreak_separator` entry to the table at the top and it is _definitely_ a `medium`. It's almost a `high` but it won't break content on load, so maybe put separator as the first medium and split block as the second.

I suggest linking both entries to this documentation section 🤔